tests/test_config_commands.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+from syncano_cli.main import cli
+from tests.base import BaseCLITest
+
+
+class ConfigCommandsTest(BaseCLITest):
+
+    def test_config(self):
+        self._add_config('test', '123')
+        result = self._list_config()
+
+        self.assertIn('test', result.output)
+        self.assertIn('123', result.output)
+
+    def test_add_config_var(self):
+        # test basic add;
+        result = self._add_config('api-key', '981xx')
+        self.assertIn('Variable `api-key` set to `981xx`.', result.output)
+
+        result = self._list_config()
+        self.assertIn('api-key', result.output)
+        self.assertIn('981xx', result.output)
+
+        # test if ERROR is displayed;
+        result = self._add_config('api-key', '981xx')
+        self.assertIn('already in config', result.output)
```

Comment:
Ok, I understand it's cross-project task. If you plan to do so - great. Just wanted to point that out.